transports/websocket/src/framed.rs

```diff
@@ -410,7 +410,7 @@ fn location_to_multiaddr<T>(location: &str) -> Result<Multiaddr, Error<T>> {
 /// The websocket connection.
 pub struct Connection<T> {
     receiver: BoxStream<'static, Result<data::Incoming, connection::Error>>,
-    sender: Pin<Box<dyn Sink<OutgoingData, Error = connection::Error> + Send>>,
+    sender: Pin<Box<dyn Sink<OutgoingData, Error = quicksink::Error<connection::Error>> + Send>>,
```

Comment:
This leaks a dependency into the public interface, correct? As the users are likely not aware of what `quicksink` is, how about doing the same trick as above converting it to an `io` error?

Reply:
No worries, the type stays private. The only API `Connection` exposes is `Stream<Item = std::io::Result<_, _>>` and `Sink<Error = std::io::Error>` plus some methods that utilise the `Sink` impl.